js/components/Table/Row.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { map } from 'lodash';
+import PropTypes from 'prop-types';
```

Comment:
I can't find where this package actually gets added to the build? It would be listed in the `package.json`, I believe, so when you run `npm install` you actually get the package. Is prop type validation working?

Reply:
it is working! Did another `npm install` though and seemed to update!

Reply:
oh wait you're right it isn't there in the file. I _thought_ it was working bc I was seeing some errors in the console of it being an object instead of an array when I was coding. I'll dig more into this!

Reply:
added !

Reply:
I think you need to add the package, like we do in Rogue https://github.com/DoSomething/rogue/blob/master/package.json#L36

Reply:
Yes, I did that! :)